components/OneXP/OneXPBanner.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="one-xp-banner">
+    <div class="cds--grid">
+      <div class="one-xp-banner__container">
+        <span class="one-xp-banner__copy" v-text="copy" />
+        <UiCta
+          class="one-xp-banner__cta"
+          kind="ghost"
+          :label="cta.label"
+          :segment="cta.segment"
+          :url="cta.url"
+        />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+interface Props {
+  copy: string;
+  cta: {
+    label: string;
+    segment: CtaClickedEventProp;
+    url: string;
+  };
+}
+
+defineProps<Props>();
+</script>
+
+<style lang="scss" scoped>
+@use "~/assets/scss/carbon.scss";
+
+.one-xp-banner {
+  background-image: linear-gradient(to right, carbon.$purple-10, carbon.$white);
+  padding: carbon.$spacing-05 0;
+
+  &__container {
+    display: inline-flex;
+    flex-direction: column;
+    gap: carbon.$spacing-03;
+
+    @include carbon.breakpoint-up(lg) {
+      flex-direction: row;
+    }
+  }
+
+  &__copy {
+    font-weight: bold;
```

Comment:
I believe this is slightly heavier, and should be 600

![Screenshot 2023-08-11 at 6 03 53 AM](https://github.com/Qiskit/qiskit.org/assets/6276074/1383b07d-61c0-4545-8bc5-e9dca275380e)

Reply:
As discussed offline, these small design details are been discussed with the design team. It is known that our codebase won't reflect exactly the Figma design due to some structural differences, but that's OK for now.